sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -542,21 +456,16 @@ export class TablesSharedKeyCredential implements TablesSharedKeyCredentialLike
     constructor(accountName: string, accountKey: string);
     readonly accountName: string;
     computeHMACSHA256(stringToSign: string): string;
-    create(nextPolicy: RequestPolicy, options: RequestPolicyOptionsLike): TablesSharedKeyCredentialPolicy;
 }
 
 // @public
-export interface TablesSharedKeyCredentialLike extends RequestPolicyFactory {
+export interface TablesSharedKeyCredentialLike {
     accountName: string;
     computeHMACSHA256: (stringToSign: string) => string;
 }
 
 // @public
-export class TablesSharedKeyCredentialPolicy extends BaseRequestPolicy {
-    constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptionsLike, credential: TablesSharedKeyCredentialLike);
-    sendRequest(request: WebResourceLike): Promise<HttpOperationResponse>;
-    signRequest(request: WebResourceLike): WebResource;
-}
+export function tablesSharedKeyCredentialPolicy(credential: TablesSharedKeyCredentialLike): PipelinePolicy;
```

Comment:
should this have a `create` prefix? the current name is a bit vague.

Reply:
Hum, we don't use `create` for any of the built-in policies today. I'm not opposed to changing this everywhere if we wanted to, but I think my hesitation is that makes it feel like it's returning a class instance instead of just an object that satisfies `PipelinePolicy`.

@bterlson do you have any naming feelings about policy functions?